### PR TITLE
ISPN-5534 CreateCacheCommand doesn't properly wait for the cache to s…

### PR DESCRIPTION
**Pull request by danberindei**
…tabilize

https://issues.jboss.org/browse/ISPN-5534

### CODE REVIEW

File: core/src/main/java/org/infinispan/commands/CreateCacheCommand.java [hunk 1]
```diff
@@ -50,43 +55,57 @@ public CreateCacheCommand(String cacheName, String cacheNameToCreate, String cac
...
+
+   protected void waitForCacheToStabilize(Cache<Object, Object> cache, Configuration cacheConfig)
+         throws InterruptedException {
+      ComponentRegistry componentRegistry = cache.getAdvancedCache().getComponentRegistry();
+      StateTransferManager stateTransferManager = componentRegistry.getStateTransferManager();
+      StateTransferLock stateTransferLock = componentRegistry.getStateTransferLock();
+      TimeService timeService = componentRegistry.getTimeService();
+
+      long endTime = timeService.expectedEndTime(cacheConfig.clustering().stateTransfer().timeout(),
+            TimeUnit.MILLISECONDS); int expectedSize = cacheManager.getTransport().getMembers().size();
```
Comment (tristantarrant) [inline core/src/main/java/org/infinispan/commands/CreateCacheCommand.java]: split the lines

Comment (danberindei) [inline core/src/main/java/org/infinispan/commands/CreateCacheCommand.java]: Oops, fixed

**Comment by wburns** [comment]
Pulling...

**Comment by wburns** [comment]
Integrated into master, thanks @danberindei !

**Pull request closed without merging**